backend/package.json

```diff
@@ -35,7 +35,10 @@
 		"@prisma/client": "^5.8.1",
 		"class-transformer": "^0.5.1",
 		"class-validator": "^0.14.1",
+		"fs-extra": "^11.2.0",
```

Comment:
In this PR, I can't find the usage of library `fs-extra`.
If you don't want to use it, please remove it from `package.json`.

Reply:
thanks.

Fix [this commit](https://github.com/yorkie-team/codepair/pull/238/commits/e0aec8c1e6b8a025521cc85379c885182ee993bb).